drake/systems/onlinePlanning/Util/simulateEuler.m

```diff
@@ -0,0 +1,31 @@
+function xtrajSim = simulateEuler(sysCl,tend,x0,dt,sys_type,xp,up)
```

Comment:
I've converted this over to use ode1 with the existing simulation pipeline but it needs the stuff in https://github.com/RobotLocomotion/drake/pull/1375